app/index_local.html

```diff
@@ -28,6 +31,11 @@
     <link
       rel="stylesheet"
       href="/static/style.css" />
+    <script>
+      function inDarkMode(){
+        return darkmode.inDarkMode;
```

Comment:
does that load the browser's default settings?

Reply:
I'm thinking of `window.matchMedia('(prefers-color-scheme: dark)').matches ? 'dark' : 'light'`